code/botlib/be_aas_route.c

```diff
@@ -105,7 +105,7 @@ void AAS_RoutingInfo(void)
 // Returns:				-
 // Changes Globals:		-
 //===========================================================================
-ID_INLINE int AAS_ClusterAreaNum(int cluster, int areanum)
+ int AAS_ClusterAreaNum(int cluster, int areanum)
```

Comment:
The changes in this file allow is to make debug builds. For debug builds only, the ID_INLINE macro caused compile errors.